Running Move unit tests
[ FAIL    ] 0x1::MissingData::missing_data
[ PASS    ] 0x1::MissingData::missing_data_captured
[ FAIL    ] 0x1::MissingData::missing_data_exact
[ FAIL    ] 0x1::MissingData::missing_data_from_other_function

Test failures:

Failures in 0x1::MissingData:

┌── missing_data ──────
│ INTERNAL TEST ERROR: UNKNOWN ERROR.
│ Test was not expected to fail, but it gave a UNKNOWN_STATUS (code 18446744073709551615) error originating in an unknown location. Rooted here
└──────────────────


┌── missing_data_exact ──────
│ INTERNAL TEST ERROR: UNKNOWN ERROR.
│ Test was not expected to fail, but it gave a UNKNOWN_STATUS (code 18446744073709551615) error originating in an unknown location. Rooted here
└──────────────────


┌── missing_data_from_other_function ──────
│ INTERNAL TEST ERROR: UNKNOWN ERROR.
│ Test was not expected to fail, but it gave a UNKNOWN_STATUS (code 18446744073709551615) error originating in an unknown location. Rooted here
└──────────────────

Test result: FAILED. Total tests: 4; passed: 1; failed: 3
